src/pages/Dashboard.js

```diff
@@ -38,15 +36,20 @@ const Dashboard = () => {
           </div>
         </Col>
         <Col lg="4">
-          <h6>NOTIFICATIONS ({notifications.length})</h6>
+          <h6>NOTIFICATIONS ({props.notifications.length})</h6>
           <div className="mt-3">
-            {notifications.map((notification, i) => {
+            {props.notifications.map((notification, i) => {
```

Comment:
Total nitpick, but if you're not using the index i, then I would remove it

Reply:
Will fix in a separate PR